Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1

```diff
@@ -212,25 +212,33 @@ function Remove-AnonymousAccessOnContainers
 
         [String]
         [Parameter(ParameterSetName = "RemediationAtStorageLevel", HelpMessage="Specifies the path to the file to be used as input for the remediation")]
-        $FilePath
+        $FilePath,
+
+        [switch]
+        [Parameter(Mandatory = $false)]
+        $AutoRemediation,
+
+        [string]
+        [Parameter(Mandatory = $false)]
```

Comment:
please make 'T' uppercase and add help text for both new params.

Reply:
Done